src/components/HeaderLink.astro

```diff
@@ -15,7 +15,7 @@ const isActive = href === pathname || href === "/" + subpath?.[0];
   class:list={[
     className,
     { active: isActive },
-    "text-whitee font-Montserrat text-lg font-normal uppercase hover:text-lightyellow",
+    "text-whitee font-Montserrat text-xl md:text-lg font-normal uppercase hover:text-lightyellow",
```

Comment:
changing a font size for the mobile version (font-size class reference https://tailwindcss.com/docs/font-size)

`text-xl` means that the text will be by default text-xl (size defined in the config under the property text-xl) 
`md:text-lg` means that from the screen size `md` (medium - the size for tablets and bigger) will have the text size lg (large - whatever the value is in the config)